boards/esp32-mh-et-live-minikit/include/periph_conf.h

```diff
@@ -72,7 +72,7 @@
  * Only I2C interface I2C_DEV(0) is used.
  *
  * @note The GPIOs listed in the configuration are only initialized as I2C
- * signals when module ```perpih_i2c``` is used. Otherwise they are not
+ * signals when module ```periph_i2c``` is used. Otherwise they are not
```

Comment:
Good catch :+1: Since it is touched anyway, please change it as following:
```suggestion
 * signals when module `periph_i2c` is used. Otherwise they are not
```